crates/rolldown_binding/src/types/binding_output_chunk.rs

```diff
@@ -1,48 +1,73 @@
 use std::collections::HashMap;
 
-use derivative::Derivative;
-use serde::Deserialize;
+use napi_derive::napi;
 
 use crate::types::binding_rendered_module::BindingRenderedModule;
 
-#[napi_derive::napi(object)]
-#[derive(Deserialize, Default, Derivative)]
-#[serde(rename_all = "camelCase")]
-#[derivative(Debug)]
+#[napi]
 pub struct BindingOutputChunk {
```

Comment:
We might need discuss about what level of structs should be apply with lazy optimization. If we just make everything lazy, it may hurt the performance of usage of users want to post process the output of rolldown. This PR shows 10% performance down for accessing all data that is passing lazily.

Reply:
Agree it. We just need to lazy fields has larger overhead.

Reply:
Wait for some benchmark for it.

Reply:
I think we can do it at new pr and easy to benchmark for it.

Reply:
Yeah. Let's merge it first. Benchmark from node.js is not usable due some problems of codspeed. We could wait.